### PR TITLE
feat: use small HTTP dependency

### PR DESCRIPTION
**Pull request by fhinkel**

### CODE REVIEW

**Comment by JustinBeckwith** [comment]
Nice!  The system and sample tests are currently down for this one, and we would need to get those passing first though :)

File: package.json [hunk 1]
```diff
@@ -94,6 +94,7 @@
     "request": "^2.88.0",
```
Comment (JustinBeckwith) [inline package.json]: Wouldn't we want to move request over to dev dependencies?

**Comment by fhinkel** [comment]
I was wondering how that tests ran so fast. No problem with waiting. How working on fixing the system tests?

File: package.json (hunk 1 above)
Comment (fhinkel) [reply package.json]: Ideally 🙈. I wrote down that `gcs-resumable-upload` depends on it. Let me check.

Comment (JustinBeckwith) [reply package.json]: Should we maybe do that library first?

**Comment by JustinBeckwith** [comment]
@fhinkel I think we have the tests worked out over here.  Can you resolve conflicts and give this another go?  Thanks!

**Pull request merged**